src/Dialogs/IconPopover.vala

```diff
@@ -0,0 +1,102 @@
+/*
+* Copyright (c) 2011 Marlin Developers (http://launchpad.net/marlin)
+* Copyright (c) 2015-2018 elementary LLC <https://elementary.io>
```

Comment:
New code should be `Copyright 2020 elementary, Inc. (https://elementary.io)` now